### DISCUSSION TITLE
[HW4] Problem 2.3

### DISCUSSION THREAD

**Discussion by emjay73** [seominjoon/kaist-ai605#165] [Q&A] [answered]
Problem 2.4 (3 points) 
Try bert-base-uncased and analyze if it makes any difference. 
What is the difference between cased and uncased in English? 
How about in Korean?

What is the meaning of the last sentence?
Do we have to train bert on Korean and check accuracy?
Or just state case sensitivity of Korean?

**Comment by seominjoon** [comment, marked as answer]
Please ignore the last sentence. I will remove this right now.
